README.md

```diff
@@ -112,6 +113,12 @@ View Inter-Blockchain Communication (IBC) transfer activity. The map traces IBC
 * [Map of Zones](https://mapofzones.com/?period=24) - [Source](https://github.com/mapofzones)
 * [Mintscan](https://hub.mintscan.io) - Interchain Explorer by Cosmostation.
 
+## Terminal Block Explorers
+
+View the block explorer on terminal without any web application (firefox, chrome...)
+
+* [cshtop](https://github.com/gsk967/cshtop) - Cosmos htop , Blocks visualizer on terminal.
```

Comment:
Could you add https://github.com/cosmos/gex before this one?

Reply:
Ok